### ISSUE TITLE
Will you provide training code in the future?

### ISSUE THREAD

**Issue by nnzs** [yehengchen/Object-Detection-and-Tracking#15]
As mentioned above, I would like to use my own training set. Can you provide training code for detection and tracking?

**Comment by yehengchen** [comment]
@nnzs If you want to train custom datasets 
First code for training a detection model - [[How to train YOLOv3 model]](https://github.com/yehengchen/Object-Detection-and-Tracking/tree/master/OneStage/yolo/yolov3)
[[cosine_metric_learning]](https://github.com/nwojke/cosine_metric_learning) - code for training a metric feature representation to be used with the deep_sort tracker.

**Comment by nnzs** [comment]
Thank you for every reply. It's very kind of you.




------------------ 原始邮件 ------------------
发件人: "Bobby Chen"<notifications@github.com>;
发送时间: 2019年9月9日(星期一) 晚上8:27
收件人: "yehengchen/Object-Detection-and-Tracking"<Object-Detection-and-Tracking@noreply.github.com>;
抄送: "嗯."<491696554@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [yehengchen/Object-Detection-and-Tracking] Will you providetraining code in the future? (#15)




@nnzs If you want to train custom datasets
 First code for training a detection model - [How to train YOLOv3 model]
 [cosine_metric_learning] - code for training a metric feature representation to be used with the deep_sort tracker.

**Comment by nnzs** [comment]
Can you provide some other test videos？thanks a lot！！！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Bobby Chen"<notifications@github.com&gt;;
发送时间:&nbsp;2019年9月9日(星期一) 晚上8:27
收件人:&nbsp;"yehengchen/Object-Detection-and-Tracking"<Object-Detection-and-Tracking@noreply.github.com&gt;;
抄送:&nbsp;"嗯."<491696554@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [yehengchen/Object-Detection-and-Tracking] Will you provide training code in the future? (#15)




@nnzs If you want to train custom datasets
 First code for training a detection model - [How to train YOLOv3 model]
 [cosine_metric_learning] - code for training a metric feature representation to be used with the deep_sort tracker.
